_partials/macos_zoom.md

```diff
@@ -0,0 +1,33 @@
+## Zoom
+
+To be able to interact when we are not in the same physical room, we will be using [Zoom](https://zoom.us/), a video conferencing tool.
+
+:warning: If you already have Zoom installed, please make sure that the version is at least **5.6**.
+
+Go to [zoom.us/download](https://zoom.us/download).
+
+Under **Zoom Client** click the **Download** button.
+
+Open the file you have just downloaded to install the app.
+
+Open the Zoom app.
+
+If you have a Mac with Apple silicon, [you are asked to install Rosetta](https://support.apple.com/en-us/HT211861). Click Install, then enter your user name and password to allow installation to proceed.
```

Comment:
This was the point mentioned on Slack, what if they already have Zoom. We should insist on the Rosetta setup here, whatever they got Zoom or not, wdyt?

Reply:
For the data setup, users will install Rosetta anyway because we are still calling the dedicated partial in the build file [here](https://github.com/lewagon/data-setup/blob/master/build.rb#L18). So even if they have Zoom already installed, Rosetta will be installed.

For the web setup, Rosetta is only needed for the Heroku CLI pretty late in the bootcamp and I'm planning to update the installation instructions [here](https://github.com/lewagon/karr/blob/master/lectures/rails/hosting-image-upload-active-storage-6.md#install-on-os-x-1) adding extra instructions to install Rosetta just in case some students didn't install it on the setup day.